pyseg/__init__.py

```diff
@@ -139,15 +139,15 @@ def _genCmdToDefineSegCondaEnv(cls):
         installationCmd += 'conda activate %s && ' % PYSEG_ENV_NAME
 
         # Install non-conda required packages
-        installationCmd += 'pip install beautifulsoup4 && '
-        installationCmd += 'pip install lxml && '
-        installationCmd += 'pip install pillow &&'
-        installationCmd += 'pip install pyfits && '
-        installationCmd += 'pip install scikit-image && '
-        installationCmd += 'pip install scikit-learn && '
-        installationCmd += 'pip install scikit-fmm && '
-        installationCmd += 'pip install scipy && '
-        installationCmd += 'pip install vtk'
+        installationCmd += 'pip install beautifulsoup4==4.9.3 && '
```

Comment:
Maybe in the near future, pyseg can become a pip package itself with its own requirements.txt and avoid all this?